src/v2/view-builder/views/PollView.js

```diff
@@ -1,53 +1,57 @@
 import { loc, View } from 'okta';
 import hbs from 'handlebars-inline-precompile';
-import { BaseForm, BaseView } from '../internals';
+import { BaseFormWithPolling, BaseView } from '../internals';
 import polling from './shared/polling';
 import { MS_PER_SEC } from '../utils/Constants';
 
 const PollMessageView = View.extend({
   template: hbs`
     <div class="ion-messages-container">
-      <p>{{i18n code="poll.form.message" 
+      <p>{{i18n code="poll.form.message"
         bundle="login" arguments="countDownCounterValue" $1="<span class='strong'>$1</span>"}} </p>
     </div>
     <div class="okta-waiting-spinner"></div>
     `
   ,
   getTemplateData() {
-    const countDownCounterValue = this.options;
     return {
-      countDownCounterValue,
+      countDownCounterValue: this.options,
     };
   },
 });
 
-const Body = BaseForm.extend(Object.assign(
+const Body = BaseFormWithPolling.extend(Object.assign(
   {
     title() {
       return  loc('poll.form.title', 'login');
     },
 
     noButtonBar: true,
-    
+
     initialize() {
-      BaseForm.prototype.initialize.apply(this, arguments);
+      BaseFormWithPolling.prototype.initialize.apply(this, arguments);
       this.startPolling();
+      this.listenTo(this.options.appState, 'change:dynamicRefreshInterval', () => {
+        const countDownCounterValue = Math.ceil(this.options.appState.getPollInterval() / MS_PER_SEC);
+        this.$el.find('.ion-messages-container span').text(countDownCounterValue);
+        this.startCountDown('.ion-messages-container span', MS_PER_SEC);
```

Comment:
looks like `startCountDown` could be better defined in PollView instead of `polling.js` as it's only used for PollView today?